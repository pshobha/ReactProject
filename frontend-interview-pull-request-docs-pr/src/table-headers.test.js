import React from 'react';
import renderer from 'react-test-renderer';
import TableHeaders from './table-headers.component.js';

describe(`<TableHeaders />`, () => {
	it(`renders some table headers`, () => {
		const tree = renderer.create(<TableHeaders />);
		expect(tree).toMatchSnapshot();
	});
});
