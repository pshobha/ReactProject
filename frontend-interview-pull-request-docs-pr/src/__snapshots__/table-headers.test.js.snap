// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableHeaders /> renders some table headers 1`] = `
<div
  className="tableHeader"
>
  <div>
    Name
  </div>
  <div>
    Date added
  </div>
  <div>
    Added by
  </div>
</div>
`;
